packages/aft/lib/src/commands/bootstrap_command.dart

```diff
@@ -79,6 +79,15 @@ const amplifyEnvironments = <String, String>{};
           // command is significantly newer/older than the embedded one.
           (pkg) => pkg.name != 'aft',
         )
+        .where(
+          // Skip bootstrapping packages which set incompatible Dart SDK constraints,
+          // e.g. packages which are leveraging preview features.
+          //
+          // The problem of packages setting incorrect constraints, for example setting
+          // `^3.0.5` when the current repo constraint is `^3.0.0` and we're running
+          // `aft` with `3.0.1` is a different issue handled by the constraints commands.
+          (pkg) => pkg.compatibleWithCurrentSdk,
```

Comment:
Should we print the packages that are ignored as a result?  In the future should someone change to a incompatible Dart SDK for whatever reason, perhaps it would be good to notify it's being ignored in bootstrap should some confusion result.

Reply:
Done 👍